.gitignore

```diff
@@ -7,7 +7,7 @@
 
 #ignore any files in config/, except those in subdirectories.
 /config/*
-!/config/*/*
+!config/**/*/
```

Comment:
This seems to be a leftover from the PR I took it from - I'll restore it to what it was previously.